GitUI/UserControls/WarningToolStripItem.cs

```diff
@@ -1,39 +1,70 @@
 ﻿using System;
+using System.Diagnostics;
```

Comment:
Can you please explain the changes to this file?

Reply:
I don't know, it was done by the original author. Let me check.